ReactAndroid/src/androidTest/java/com/facebook/react/tests/CatalystUIManagerTestCase.java

```diff
@@ -102,7 +104,7 @@ public void run() {
 
   public void testFlexUIRendered() {
     FrameLayout rootView = createRootView();
-    jsModule.renderFlexTestApplication(rootView.getId());
+    jsModule.renderFlexTestApplication(reactTagFrom(rootView));
```

Comment:
Should this be `reactTagFor` or just `getReactTag`?

It returns a tag *for* a view, not a tag *from* a view, correct?